pkg/daemon/daemon_test.go

```diff
@@ -105,15 +117,49 @@ var _ = Describe("Config Daemon", func() {
 
 		vars.UsingSystemdMode = false
 		vars.NodeName = "test-node"
+		vars.Namespace = "sriov-network-operator"
 		vars.PlatformType = consts.Baremetal
 
-		kubeClient := fakek8s.NewSimpleClientset(&FakeSupportedNicIDs, &SriovDevicePluginPod)
-		client := fakesnclientset.NewSimpleClientset()
+		err = sriovnetworkv1.AddToScheme(scheme.Scheme)
+		Expect(err).ToNot(HaveOccurred())
+		kClient := kclient.NewClientBuilder().WithScheme(scheme.Scheme).WithRuntimeObjects(&corev1.Node{
+			ObjectMeta: metav1.ObjectMeta{Name: "test-node"}},
+			&sriovnetworkv1.SriovNetworkNodeState{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-node",
+					Namespace: vars.Namespace,
+				}}).Build()
+
+		FakeSupportedNicIDs := corev1.ConfigMap{
```

Comment:
`FakeSupportedNicIDs` and `SriovDevicePluginPod` are already defined on lines L35 and L46. I think we can keep them or these, no preference.

Reply:
you are right I miss that thanks!